docs/lang/articles/advanced/sparse.md

```diff
@@ -4,7 +4,69 @@ sidebar_position: 3
 
 # Sparse computation
 
-The LLVM backends (CPU/CUDA) and the Metal backend offer the full functionality of spatially sparse computation in Taichi.
+## Motivation
+
+High-resolution 2D/3D grids are often needed in large-scale spatial computation, especially physical simulation.
+However, they tend to take a huge amount of memory space and computation.
+
+While a programmer may allocate large huge dense grids to store spatial data (especially physical quantities such as a density field),
+oftentimes he or she only cares about a small fraction of this dense grid, the rest being ambient space (air).
```

Comment:
It has become increasing popular in technical documentation to use the singular *they* as the general pronoun (e.g., [as in Google docs style guide](https://developers.google.com/style/pronouns)). I wonder if we shall also adopt this to make our wording fully inclusive to gender identities. [The singular *they* is used exactly the same as the regular plural *they* where third-person singular is not applied](https://en.wikipedia.org/wiki/Singular_they).

Reply:
I think that is a great point and we should defo do it, but requires a huge effort to change all of the pronouns across our documentation, so maybe we could do it in a standalone PR? And while we are still on the topic of inclusion, our default branch is still not updated to `main` 😅

Reply:
😂 Fortunately we haven't had any pronouns in the current documentation, if I've used our docs search correctly. Then the effort would be simply on writing the updated documentations.

Reply:
Sounds great and I will adopt singular *they* here. Today I learn something!